src/lib/pushNotifications.ts

```diff
@@ -19,6 +19,15 @@ const sendPushNotification = async () => {
   }
 
   const messages: any = [];
+
+  const completeHabitMessage = "\n\nDon't forget to complete a habit today!";
+  const notificationMessages = [
+    '"A journey of 1000 miles starts with a single step."' + completeHabitMessage,
+    '"We are what we repeatedly do. Excellence, then, is not an act, but a habit."' + completeHabitMessage,
+    '"Motivation is what gets you started. Habit is what keeps you going."' + completeHabitMessage,
+  ];
+  var randomMessage = notificationMessages[Math.floor(Math.random() * notificationMessages.length)];
```

Comment:
I approved already but could probably change this to a let

Reply:
Will change before merging.